CMakeLists.txt

```diff
@@ -7,10 +7,15 @@ endif()
 
 project(xatlas)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
-if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 7.0)
-    # avoid the "‘nodiscard’ attribute directive ignored" warning
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-attributes")
+if (WIN32)
+    # clang on windows uses visual studio standard library, which uses c++14 features
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
+else()
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 7.0)
+        # avoid the "‘nodiscard’ attribute directive ignored" warning
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-attributes")
+    endif()
```

Comment:
Hm, could we change to c++14 everywhere?

Reply:
Can we do that in future PR?

Reply:
This is fine. I agree with Jan that we could update everywhere (not necessarily now).